yt_dlp/extractor/naver.py

```diff
@@ -326,39 +328,41 @@ def _extract_replay(self, show_id, replay_id):
         }, self._extract_video_info(replay_id, vod_info['video_id'], in_key))
 
     def _extract_show_replays(self, show_id):
-        page = 0
+        page_size = 15  # page_size is now a fixed unchangeable value
```

Comment:
imo inlining is better

Reply:
The whole of _extract_show_replays and _extract_show_highlights needs some refactoring. But beyound the scope of this PR